Running ogip_check with
    input = inputs/fermi_lat_bcf_edisp_calfile.fits
    otype = BLAH
    logfile = <open file '<stdout>', mode 'w' at 0x10f50f150>
    verbosity = 2
    dtype = None
    vonly = False
    meta_key = default
Attempting to open file inputs/fermi_lat_bcf_edisp_calfile.fits
File opened successfully.
Running basic FITS verification with external call to ftverify.
Nothing from ftverify, trying with astropy.io.fits.verify(option='exception')

Checking file as type BLAH
Currently defined for
Type = TIMING, SPECTRAL, CALDB, ARF, RMF, or IMAGE.
WARNING:  do not recognize OGIP type BLAH
